capz/run-capz-e2e.sh

```diff
@@ -118,9 +118,12 @@ create_cluster(){
 }
 
 apply_workload_configuraiton(){
-    # A patch is needed to tell kube-proxy to use CI binaries.  This could go away once we have build scripts for kubeproxy HostProcess image.
-    kubectl apply -f "${CAPZ_DIR}"/templates/test/ci/patches/windows-kubeproxy-ci.yaml
-    kubectl rollout restart ds -n kube-system kube-proxy-windows
+    # Only patch up kube-proxy if $WINDOWS_KPNG is unset
+    if [[ -z "$WINDOWS_KPNG" ]]; then
```

Comment:
need to default this, the tests are failing

```
WINDOWS_KPNG: unbound variable
```

How is the latest version being built for kpng image?  I guess we don't have it tied directly to a kubernetes version right now?

Reply:
We just pull from the latest image in the K8s staging repo...

Reply:
updated